.gitignore

```diff
@@ -7,3 +7,4 @@ cmd/loki/loki
 cmd/promtail/promtail
 /loki
 /promtail
+.idea/
```

Comment:
Whats this?  Probably belongs in ~/.gitignore and then do `git config --global core.excludesfile $HOME/.gitignore`